internal/broker/broker_test.go

```diff
@@ -170,17 +171,17 @@ func TestGetAuthenticationModes(t *testing.T) {
 
 		wantErr bool
 	}{
-		// Auth Session
+		// Authentication session
 		"Get_device_auth_qr_if_there_is_no_token":                      {},
 		"Get_newpassword_if_already_authenticated_with_device_auth_qr": {secondAuthStep: true},
 		"Get_password_and_device_auth_qr_if_token_exists":              {tokenExists: true},
 
 		"Get_only_password_if_token_exists_and_provider_is_not_available":                {tokenExists: true, providerAddress: "127.0.0.1:31310", unavailableProvider: true},
 		"Get_only_password_if_token_exists_and_provider_does_not_support_device_auth_qr": {tokenExists: true, providerAddress: "127.0.0.1:31311", deviceAuthUnsupported: true},
 
-		// Passwd Session
-		"Get_only_password_if_token_exists_and_session_is_passwd":                      {sessionMode: "passwd", tokenExists: true},
-		"Get_newpassword_if_already_authenticated_with_password_and_session_is_passwd": {sessionMode: "passwd", tokenExists: true, secondAuthStep: true},
+		// Change password session
+		"Get_only_password_if_token_exists_and_session_is_passwd":                      {sessionMode: sessionmode.ChangePassword, tokenExists: true},
+		"Get_newpassword_if_already_authenticated_with_password_and_session_is_passwd": {sessionMode: sessionmode.ChangePassword, tokenExists: true, secondAuthStep: true},
```

Comment:
shouldn’t we have a test case with the old string there too?

Reply:
good point! done